tests/integration/test_logging.py

```diff
@@ -0,0 +1,45 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright 2023 Canonical Ltd.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License version 3 as published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import logging
+import sys
+import textwrap
+from pathlib import Path
+
+from craft_parts import main
+
+parts_yaml = textwrap.dedent(
+    """
+    parts:
+      foo:
+        plugin: nil
+        build-packages: [libc6]
+        stage-packages: [hello]
```

Comment:
no build-snaps to not make it slow?

Reply:
maybe pyright, it will most likely also already be installed :-)

Reply:
> no build-snaps to not make it slow?

That, and also to be able to run the test locally as a regular user. I'll try pyright